looker_style_guide.md

```diff
@@ -0,0 +1,170 @@
+# LookML Style Guide
+
+
+## Structure of our LookML project
+When structuring our LookML project, our views' folders should loosely reflect the structure of our dbt project with the name of the views representing our business units. For example, we have our core business units (e.g. customers) and our dbt Cloud business units (e.g. accounts, cloud projects, etc) which are similar to our [marts folder](https://github.com/fishtown-analytics/internal-analytics/tree/main/models/marts).
+
+Think of views as raw ingredients that are in the vegetable/fruit aisles in a grocery store. They are ingredients that can be combined and packaged into a product that fits our customers' needs. These "packaged items" are explores. More on that below.
+
+```
+fishtown project
+├── core
+│   └── customers.view
+├── csm
+│   └── sprints.view
+├── dbt_cloud
+│   ├── cloud_accounts.view
+│   ├── cloud_ide_sessions.view
+│   └── cloud_projects.view
+├── dbt_core
+│   ├── dbt_doc_viewer_activity.view
+│   └── dbt_projects.view
+├── finance
+│   ├── quickbook_invoices.view
+│   └── subscription_transactions.view
+└── fishtown.model
+```
+
+Explores should be optimized for the business stakeholder where we're getting the best represented business unit to run our analyses off of. This means that explores should be organized based on department or category of the business unit. For example, we have several Slack views, but maybe they all join under a singular `Slack Messages` explore, which should fall under the `Community` category.
+
+Building off of our above analogy, explores are the packaged items that can combine ingredients (views) to suit a customer's needs. We can utilize various views to create different explores.
+
+
+## Views
+
+#### Structure
+* A view's name should represent the business unit (i.e. the level of granularity of the table). For example, if we're taking `fct_intercom_conversations`, then the view name should be: `intercom_conversations`.
+* The `sql_table_name` should always have the [user attribute feature](https://blog.getdbt.com/how-to-integrate-dbt-and-looker-with-user-attributes/).
+* Dimensions and measures should be organized by group label (Note: for dimensions, date groupings should always be last)
+
+#### Dimensions & Measures
+* Dimensions and measures should be ordered as (if fields are applicable): name, label, group_label, description, hidden, type, sql, value_format_name, filter
+* Both should have a description
+* We prefer `value_format_name` over `value_format`
+* Measures should reference the dimension (e.g. ${order_total} over ${TABLE}.order_total)
+
+#### Example
+
+```
+view: intercom_conversations {
+  sql_table_name:
+  -- if prod -- analytics.analytics.fct_intercom_conversations
+  -- if dev -- analytics.{{_user_attributes['dbt_schema']}}.fct_intercom_conversations
+  ;;
+
```

Comment:
As @dave-connors-3 pointed out , I think this example is just missing a # ==== Dimensions heading!